modules/ROOT/pages/to-change-raml-version.adoc

```diff
@@ -24,36 +24,29 @@ NOTE: New asset versions count toward your organization's xref:to-create-an-asse
 
 == Deploy a New API Asset
 
-You can deploy an API asset directly into Exchange using the *New asset* button. In the *Publish a new asset* window, the Advanced section enables you to specify the version of the asset in the *Version* field, and the version of the API specification in the *API version* field.
+You can deploy an API asset directly into Exchange using the *Publish new asset* button. In the *Publish a new asset* window, the Advanced section enables you to specify the version of the asset in the *Version* field, and the version of the API specification in the *API version* field.
 
-image::ex-publish-a-new-asset.png[]
 
 Only change the API version when an updated API specification has changes that are not backward compatible from the new version to the old version.
 
 == Change RAML or OAS Version
 
-For RAML and OAS API specifications within Exchange, you can change the version number from within Exchange using the *Add new version* button in the Exchange user interface.
+For RAML and OAS API specifications within Exchange, change the version number from within Exchange using the *Add version* button. 
 
-image::ex-add-new-version.png[]
+. From the asset details pane, click *Add version*. 
+. (Required) Select a *Method* for the API specification.
+. (Required) From *File upload* click *Choose file*. For OAS, choose an OAS file with the .yaml or .json file types. For RAML, choose a RAML file with the .raml file type. Zip files are also accepted with the file to upload residing in the zip file’s root directory.
+. Specify the *Main file* if the uploaded file is a zip. Otherwise, Exchange uses the same file name as the file you upload. 
```

Comment:
```suggestion
. Specify *Main file* if the uploaded file is a ZIP file. Otherwise, Exchange uses the same file name as the name of the file you upload. 
```